ext/native/thin3d/VulkanQueueRunner.cpp

```diff
@@ -2,6 +2,8 @@
 #include "VulkanQueueRunner.h"
 #include "VulkanRenderManager.h"
 
+// TODO: This is only enough for 4x render resolution / 4x texture upscale for debugger.
```

Comment:
Maybe a largerReadbackBuffer_?  Or maybe even not keep it around... hmm.

-[Unknown]

Reply:
Or ideally, resize it on demand. Don't want to make a too crazy allocation when it's not needed..